app/routes/register-your-interest/check-your-answers-and-register-your-interest.js

```diff
@@ -0,0 +1,71 @@
+const session = require('../../session')
+const sessionKeys = require('../../session/keys')
+const urlPrefix = require('../../config/index').urlPrefix
+const callChargesUri = require('../../config/index').callChargesUri
+const ruralPaymentsEmail = require('../../config/index').ruralPaymentsEmail
+
+const PATH = `${urlPrefix}/register-your-interest/check-your-answers-and-register-your-interest`
+
+module.exports = [
+  {
+    method: 'GET',
+    path: PATH,
+    options: {
+      auth: false,
+      handler: async (request, h) => {
+        const rows = [
+          {
+            key: { text: 'Customer reference number' },
+            value: { html: session.getRegisterYourInterestData(request, sessionKeys.registerYourInterestData.crn) },
+            actions: {
+              items: [
+                {
+                  href: `${urlPrefix}/register-your-interest/enter-your-crn`,
+                  text: 'Change',
+                  visuallyHiddenText: 'Change your CRN'
+                }
+              ]
+            }
+          },
+          {
+            key: { text: 'Single business identifier number' },
+            value: { html: session.getRegisterYourInterestData(request, sessionKeys.registerYourInterestData.sbi) },
+            actions: {
+              items: [
+                {
+                  href: `${urlPrefix}/register-your-interest/enter-your-sbi`,
+                  text: 'Change',
+                  visuallyHiddenText: 'Change your SBI'
+                }
+              ]
+            }
+          },
+          {
+            key: { text: 'Email address' },
+            value: { html: session.getRegisterYourInterestData(request, sessionKeys.registerYourInterestData.emailAddress) },
+            actions: {
+              items: [
+                {
+                  href: `${urlPrefix}/register-your-interest/enter-your-email-address`,
+                  text: 'Change',
+                  visuallyHiddenText: 'Change your email address'
+                }
+              ]
+            }
+          }
+        ]
+        return h.view('register-your-interest/check-your-answers-and-register-your-interest', { rows })
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: PATH,
+    options: {
+      auth: false,
+      handler: async (request, h) => {
+        return h.redirect('registration-complete', { callChargesUri, ruralPaymentsEmail })
```

Comment:
might want to clear data from the session - noticed on demo it persists after a successful registration of interest

Reply:
done